src/store/adminUsers/actions.js

```diff
@@ -34,6 +55,14 @@ export const createUserSuccess = createAction(CREATE_SUCCESS, (user) => user);
 export const setUsers = createAction(SET_USERS, (users) => users);
 export const appendUsers = createAction(APPEND_USERS, (users) => users);
 export const changeSearchInput = createAction(CHANGE_SEARCH_INPUT, (value) => value);
+export const passwordLengthError = createAction(PASSWORD_LENGTH_ERROR, (value) => value);
+export const passwordEqualError = createAction(PASSWORD_EQUAL_ERROR, (value) => value);
+export const getUserTypeValue = createAction(FORM_USER_TYPE_INPUT_VALUE, (object) => object.target.value);
+export const getUsernameValue = createAction(FORM_USERNAME_INPUT_VALUE, (value) => value);
+export const getPasswordValue = createAction(FORM_PASSWORD_INPUT_VALUE, (value) => value);
+export const getUserDescriptionValue = createAction(FORM_DESCRIPTION_INPUT_VALUE, (value) => value);
+export const getPasswordRepeatValue = createAction(FORM_PASSWORD_REPEAT_INPUT_VALUE, (value) => value);
```

Comment:
Why do these actions start with `get` ?

Reply:
do you prefer another start? Like `change`?

Reply:
Yes, change is better